parts/linux/cloud-init/artifacts/cse_config.sh

```diff
@@ -390,6 +390,16 @@ ensureKubelet() {
     {{end}}
 }
 
+ensureLabels() {
```

Comment:
if we're not going to merge with ensureLabelNodes below we should leave a commetn explaining what the difference is.

Reply:
The both interact with labels. However, I feel like they have different responsibilities, so I'm personal leaning towards separate services, but would be curious if others think differently. 

I'll add a comment :)

Reply:
Added two comments. Curious if they are explanative enough for a reader. What do you think?

Reply:
Removing ensureLabelNodes and associated files/code because of Tony's opinions